app/main/views/marketplace.py

```diff
@@ -123,50 +132,137 @@ def get_brief_by_id(framework_framework, brief_id):
+        index=index,
+        doc_type=doc_type,
+        **build_search_query(
+            updated_request_args if updated_request_args else clean_request_query_params,
+            filters.values(),
+            content_manifest,
+            lots_by_slug
+        )
+    )
+
+    # Convert the values of certain attributes to their label counterparts
+    content = content_loader.get_manifest(framework['slug'], 'briefs_search_filters')
+    for brief in search_api_response['documents']:
+        if brief.get('specialistRole'):
+            brief['specialistRole'] = content.summary(brief).get_question('specialistRole').value
+        brief['location'] = content.summary(brief).get_question('location').value
+
+    search_results_obj = SearchResults(search_api_response, lots_by_slug)
 
-    if not frameworks:
-        abort(404, "No framework {}".format(framework_framework))
-
-    # disabling csrf protection as this should only ever be a GET request
-    form = BriefSearchForm(request.args, frameworks=frameworks, data_api_client=data_api_client, csrf_enabled=False)
-    if not form.validate():
-        abort(404, "Invalid form data")
-
-    api_result = form.get_briefs()
-
-    briefs = [{
-        "data": brief,
-        "content": content_loader.get_manifest(brief['frameworkSlug'], 'display_brief').filter(brief)
-    } for brief in api_result["briefs"]]
-
-    links = api_result["links"]
-
-    api_prev_link_args = parse_link(links, "prev")
-    prev_link_args = None
-    if api_prev_link_args:
-        prev_link_args = request.args.copy()
-        prev_link_args.setlist("page", api_prev_link_args.get("page") or ())
-
-    api_next_link_args = parse_link(links, "next")
-    next_link_args = None
-    if api_next_link_args:
-        next_link_args = request.args.copy()
-        next_link_args.setlist("page", api_next_link_args.get("page") or ())
-
-    return render_template('search/briefs.html',
-                           framework=frameworks[-1],
-                           form=form,
-                           filters=form.get_filters(),
-                           filters_applied=form.filters_applied(),
-                           briefs=briefs,
-                           lot_names=tuple(label for id_, label in form.lot.choices),
-                           prev_link_args=prev_link_args,
-                           next_link_args=next_link_args,
-                           briefs_count=api_result.get("meta", {}).get("total", None),
-                           )
+    pagination_config = pagination(
+        search_results_obj.total,
+        current_app.config["DM_SEARCH_PAGE_SIZE"],
+        get_page_from_request(request)
+    )
+
+    search_summary = SearchSummary(
+        search_api_response['meta']['total'],
+        clean_request_query_params.copy(),
+        filters.values(),
+        lots_by_slug
+    )
+
+    category_filter_group = filters.pop('categories') if 'categories' in filters else None
```

Comment:
Feels like potentially we _do_ have some logic here that could be abstracted out between briefs and services filtering - which would be something to tackle in that other ticket, if there's time.